### ISSUE TITLE
Incompatibility issue with Nest 5.7.4

### ISSUE THREAD

**Issue by focux** [fwoelffel/nest-bull#82]
Just for the record, there are some incompatibility issues with Nest 5.7.4. Specifically, the **ModulesContainer** class export is missing in the nestjs core, this is the error I'm getting:

```
node_modules/nest-bull/dist/bull.explorer.d.ts:2:10 - error TS2305: Module /node_modules/@nestjs/core"' has no exported member 'ModulesContainer'.

import { ModulesContainer, ModuleRef } from '@nestjs/core';
```

 For version > 6.0 I see that the export is there. In case is intended to not support versions lower than 6, I think you should put that in the readme.

Thank you for such a great package!

**Comment by fwoelffel** [comment]
Lots of application are still depending on Nest 5. I'll try to find a fix to this.
Thanks for reporting! :robot:

**Comment by fwoelffel** [comment]
Sadly this module heavily relies on the `ModulesContainer` since its 0.6.2 release. I haven't found any workaround so I'm left with no other choice than deprecating Nest 5. To be honest, this was going to happen sooner or later.
Have a look at https://docs.nestjs.com/migration-guide to migrate your Nest application to the latest version.

**Comment by fwoelffel** [comment]
Closed with 3fd130e49a03d4602832090f593b3ad89a7c01d2
